USAGE_android.md

```diff
@@ -502,6 +502,21 @@ adb pull /sdcard/Download/gfxrecon_trace.gfxr
 
 This will download the file to the current directory.
 
+### Trimmed Captures
+
+Trimmed captures are created when GFXR is configured to start capturing at some later
+time in execution.
```

Comment:
It's hard to write a tight definition of a trimmed capture. What about changing the above to:

"...configured to not start capturing at the first API command that it observes but later, potentially after many frames of rendering have been skipped. In order to allow replay to also skip to the trim point capture writes a state dump at the start of the capture file, which replay can load."

_Sorry, I'm writing on a merged PR. Leaving these comments here for me to turn into a PR on top of this PR later._